applications/feedback_sentiment_analyzer/SPECIFICATION.md

```diff
@@ -0,0 +1,224 @@
+# FSA Tech spec
+
+This document contains the technical specifications for Feedback Sentiment Analyzer (FSA), a sample application that showcases AWS services and SDKs.
+
+It explains:
+* Application inputs and outputs
+* Underlying AWS components and their configurations
+* Key cross-service integration details
+
+### Table of Contents
+  * [Inputs](#inputs)
+  * [Outputs](#outputs)
+  * [Diagram](#diagram)
+  * [Input routing](#input-routing)
+  * [Step Function configuration](#step-function-configuration)
+    + [State machine Lambda functions](#state-machine-lambda-functions)
+      - [Diagram](#diagram-1)
+      * [Function inputs and outputs](#function-inputs-and-outputs)
+        + [ExtractText](#extracttext)
+        + [AnalyzeSentiment](#analyzesentiment)
+        + [TranslateText](#translatetext)
+        + [SynthesizeAudio](#synthesizeaudio)
+  * [GetFeedback Lambda function](#getfeedback-lambda-function-1)
+  * [Processing S3 events with EventBridge](#processing-s3-events-with-eventbridge)
+  * [Managing items in DynamoDB](#managing-items-in-dynamodb)
+  * [Other FSA material](#other-fsa-material)
+
+---
+
+## Inputs
```

Comment:
These inputs and outputs don't seem very helpful to me as their own section. The lambda specs definitely need input shape and output shape, but the content here might fit better within and end-to-end description of how the app works.

Reply:
User actions and application behavior.